models/business-unit/src/company/builder.spec.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import type { TBusinessUnit } from '../types';
+import * as BusinessUnit from './index';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TBusinessUnit, TBusinessUnit>(
+      'default',
+      BusinessUnit.random(),
+      expect.objectContaining({
+        id: expect.any(String),
+        version: expect.any(Number),
+        key: expect.any(String),
+        status: expect.any(String),
+        stores: [],
+        storeMode: 'Explicit',
+        unitType: 'Company',
```

Comment:
Not sure I know how these tests are set up. Is the 3rd param the expect?

When you call createBuilderSpec and don't pass in anything specific to a company (assuming the first 2 params are what you are asking it to build), how do you get a unitType of "company" back? I noticed the test for division is identical, but the unit Type is "Division".